src/main/java/com/toptrumps/core/card/Card.java

```diff
@@ -11,11 +11,11 @@
 
     private String name;
     private ArrayList<Attribute> attributes;
+    private ArrayList<Attribute> highestAttributes;
```

Comment:
We don't need a member variable for this. You could do the whole operation inside the ```selectAttribute()``` AI method.